_includes/manuals/1.12/3.2.3_installation_scenarios.md

```diff
@@ -1,6 +1,20 @@
 
 The Foreman installer can accommodate more complex, multi-host setups when supplied with appropriate parameters.
 
+#### Using an external database server
+
+Per default foreman-installer will install a PostgreSQL database server onto the Foreman host and create its database. An external MySQL or PostgreSQL database server with an already created, empty database can be used with the following arguments:
```

Comment:
Indeed, added. https://github.com/theforeman/puppet-foreman/issues/473 is related.